holidays/locale/de/LC_MESSAGES/PL.po

```diff
@@ -0,0 +1,115 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# Poland holidays de localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.69\n"
+"POT-Creation-Date: 2025-03-13 19:31+0200\n"
+"PO-Revision-Date: 2025-03-13 19:31+0200\n"
+"Last-Translator: pyniuX <krzysiek01pk@gmail.com>\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingua 4.15.0\n"
+"X-Generator: Poedit 3.5\n"
+
+#. New Year's Day.
+msgid "Nowy Rok"
+msgstr "Neujahr"
+
+#. Epiphany.
+msgid "Święto Trzech Króli"
+msgstr "Heilige Drei Könige"
+
+#. Candlemas.
+msgid "Oczyszczenie Najświętszej Marii Panny"
+msgstr "Darstellung des Herrn"
+
+#. Easter Sunday.
+msgid "Niedziela Wielkanocna"
+msgstr "Ostersonntag"
+
+#. Easter Monday.
+msgid "Poniedziałek Wielkanocny"
+msgstr "Ostermontag"
+
+#. National Day.
+msgid "Święto Państwowe"
+msgstr "Tag der Arbeit"
+
+#. National Day of the Third of May.
+msgid "Święto Narodowe Trzeciego Maja"
+msgstr "Nationalfeiertag am 3. Mai"
+
+#. National Victory and Freedom Day.
+msgid "Narodowe Święto Zwycięstwa i Wolności"
+msgstr "Nationalfeiertag des Sieges und der Freiheit"
+
+#. Ascension Day.
+msgid "Wniebowstąpienie Pańskie"
+msgstr "Christi Himmelfahrt"
+
+#. Pentecost.
+msgid "Zielone Świątki"
+msgstr "Pfingsten"
+
+#. Whit Monday.
+msgid "Drugi dzień Zielonych Świątek"
+msgstr "Pfingstmontag"
+
+#. Corpus Christi.
+msgid "Dzień Bożego Ciała"
+msgstr "Fronleichnam"
+
+#. Saints Peter and Paul Day.
+msgid "Uroczystość Świętych Apostołów Piotra i Pawła"
+msgstr "Fest der Apostelfürsten Petrus und Paulus"
+
+#. National Day of Rebirth of Poland.
+msgid "Narodowe Święto Odrodzenia Polski"
+msgstr "Nationalfeiertag der polnischen Wiedergeburt"
```

Comment:
```suggestion
msgstr "Nationalfeiertag der Wiedergeburt Polens"
```
As far as I understand it, this refers to rebirth of Poland (the country), not Poles (the nation) - "Odrodzenia Polski", not "Odrodzenia Polskiego".